app/components/CapitalProjectsList/CapitalProjectsDrawer.tsx

```diff
@@ -0,0 +1,89 @@
+import { Box, Flex, Heading } from "@nycplanning/streetscape";
+import {
+  Drawer,
```

Comment:
If you upgrade the installed Streetscape version to `^0.10.4`, you can get these component from there instead of importing from Chakra directly.